Directory.Build.targets

```diff
@@ -22,5 +22,8 @@
     </RestoreSources>
   </PropertyGroup>
 
+ <ItemGroup Condition="'$(MSBuildProjectExtension)' == '.csproj'">
+   <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="3.7.0" />
```

Comment:
Does version 3.7.0 have what we need? I thought we needed to use a pre-release version of the package.

Reply:
Synced offline that this package version works.